lib/Dialect/SV/Transforms/SVExtractTestCode.cpp

```diff
@@ -392,9 +472,45 @@ struct SVExtractTestCodeImplPass
     // erase old operations of interest
     for (auto op : roots)
       op->erase();
+
+    // Erase any instances that were extracted, and their forward dataflow.
+    for (auto *op : opsToErase) {
+      op->dropAllUses();
+      op->erase();
+    }
+
     // Move any old modules that are test code only to the test code area.
     maybeMoveToTestCode(module, testBenchDir, bindFile, instanceGraph);
   }
+
+  void maybeMoveExtractedModule(hw::InstanceGraph &instanceGraph,
```

Comment:
Is this function ever called? 🤔

Reply:
hmm must have been a rebase issue, it should have been! thanks

Reply:
fixed in [091573c](https://github.com/llvm/circt/pull/3522/commits/091573ca45cd024fcc09b01bc9c907326452500b)